### PR TITLE
Create infobox plugin

### LINKED ISSUE #7: new plugin: infobox
As discussed in the call with @schul-cloud last week, a plugin for an infobox is desired.

We thought about two possible design ideas that we would like to discuss. As we need the `Slate` Plugin for editing capabilities these two ideas came to our mind:
1) Wrap the `Slate` Plugin in our `infobox` plugin (i.e. `<infobox><Slate /></infobox>`)
1) Extend the `Slate`plugin with an extra button to toggle the `infobox` style

How do you feel about these suggestions?

@jprberlin @Knorrke

### PR DESCRIPTION
**Pull request by FWirtz**
Here you go @Knorrke, please let us know what you think. 😊 
Simple, green-bordered layout plugin.
 
image

Fixes #7

### CODE REVIEW

**Comment by FWirtz** [comment]
We added our own `postcss.colors.js` and `postcss.config.js` to be able to build. We'd love to improve that but don't really know how @Knorrke

**Comment by jprberlin** [comment]
Can the Color be chosen in the settings? If not this would be a nice next iteration.

**Comment by Knorrke** [comment]
Hey, @FWirtz thanks for the PullRequest! I forgot about the `postcss.config.js`, great you got it to work! I will change this a little bit, because we don't need the `postcss.colors.js`. Also i forgot to exclude the bundled `lib` files from git, they aren't needed in the repo (did this wrong in our spoiler plugin too).

Would be great if you could create new branches in your fork for pull requests, so i can push some changes to it without breaking your master branch ;)
I'll add some coments in the files, but i think its configuration things that i can fix myself.

I like the suggestion by @jprberlin to have the color customizable, i think this would be a good way to improve the plugin further.

File: example/package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "font-awesome": "^4.7.0",
     "ory-editor": "^0.2.24",
+    "ory-editor-plugins-infobox": "file:../plugins/infobox",
```
Comment (Knorrke) [inline example/package.json]: Lerna should handle the linking so it doesn't have to be set absolute. To do this just set the version here to `0.0.0`, the correct one will then be resolved by lerna.
The package has to be defined in workspaces in the `package.json` of the whole repo.
(I will change this, so just fyi)

File: plugins/infobox/src/index.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+const InfoboxPlugin = ({ children }) => (
+    <div className="ory-editor-plugins-infobox">
+        { children }
+    </div>
+)
+
+export default ({ defaultPlugin }) => ({
+    Component: InfoboxPlugin,
+    IconComponent: <InfoIcon />,
+    name: 'example/layout/infobox',
```
Comment (Knorrke) [inline plugins/infobox/src/index.js]: You could change this to something like `schul-cloud/layout/infobox` or `serlo/layout/infobox` if you like. Not that important atm, but might be more descriptive.

File: plugins/postcss.config.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+module.exports = (ctx) => ({
+  map: ctx.options.map,
+  parser: ctx.options.parser,
+  plugins: {
+    'postcss-import': { root: ctx.file.dirname },
+    'postcss-nested': { },
+    'postcss-cssnext': {
```
Comment (Knorrke) [inline plugins/postcss.config.js]: We can remove this for now. We don't have that many colors, so we don't need variables atm.

**Comment by Knorrke** [comment]
Code is fine apart from the config changes above. Thank you! :+1: Did ypu get along with the documentation, or are there things to improve? Your feedback would help us a lot!

**Comment by FWirtz** [comment]
Closed in favour of #11

**Pull request closed without merging**